src/system/sys_profiler.hh

```diff
@@ -342,6 +353,13 @@ public:
      */
     void add_child(int child_index, const Timer &child);
 
+    /**
+     * Set memory monitoring flag to true
+     */
+    inline void set_turn_off_memory_monitoring() {
```

Comment:
Prejmenovat na `start_memory_monitoring` a zde je 
1. nutno otestovat zda je globalne memory monitoring zapnuty
2. precist stav pameti z PETSC, to co je nyni v Timer::start().
3. nastavit memory_monitoring_on_